plugins/module_utils/azure/client.py

```diff
@@ -0,0 +1,224 @@
+import json
+from typing import Any, Dict, Tuple
+import copy
+import uuid
+
+from ansible.module_utils.basic import to_native
+from ansible_collections.cloud.pravic.plugins.module_utils.azure.credentials import AzureCredentials
+
+try:
+    from msrestazure.tools import resource_id
+    from msrestazure.azure_configuration import AzureConfiguration
+    from msrest.service_client import ServiceClient
+    from msrestazure.azure_exceptions import CloudError
+    from msrest.pipeline import ClientRawResponse
+    from msrest.polling import LROPoller
+    from msrestazure.polling.arm_polling import ARMPolling
+
+except ImportError:
+    pass
+
+from ansible_collections.cloud.pravic.plugins.module_utils.resource import CloudClient
+from ansible_collections.cloud.pravic.plugins.module_utils.exception import CloudException
+
+
+def dict_merge(a, b):
+    '''recursively merges dicts. not just simple a['key'] = b['key'], if
+    both a and b have a key whose value is a dict then dict_merge is called
+    on both values and the result stored in the returned dictionary.'''
+    if not isinstance(b, dict):
+        return b
+    result = copy.deepcopy(a)
+    for k, v in b.items():
+        if k in result and isinstance(result[k], dict):
+            result[k] = dict_merge(result[k], v)
+        else:
+            result[k] = copy.deepcopy(v)
+    return result
```

Comment:
It looks like this is copy/paste from `ansible.module_utils.common.dict_transformations`. Can we just import it, instead?